transcom/mymove
# B 18404 sit service item info population int (DO NOT MERGE BEFORE B-18885)

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18404)

## Summary
Adjusts service item detail display as well as the service item model to payload handler. This allows the removal of redundant information and allows the UI to be to the customers desire (They attached screenshots in the ticket).

### Upstream
18885 is marked as an 'upstream' because SIT delivery miles were adjusted, and this PR adds SIT delivery miles to the service item payload.

### How to test

1. Login as Prime
2. Select a move
3. Create a new HHG shipment
4. Add a new origin SIT service item on that new shipment
5. Add a new destination SIT service item on that new shipment
6. Make note of your shipment information and your move code
7. Login as TOO
8. Navigate to your move
9. Approve your shipment
10. Review requested service items and their information
11. Optionally, adjust database values of these service items to see the changes reflect. Or, alternatively, navigate to existing seed data moves such as move locator code `ORGSIT`

## Screenshots

### Domestic Origin SIT
![image](https://github.com/transcom/mymove/assets/136814362/7a7a1c21-e5e9-4e16-8196-0998014e2980)

### Domestic Destination SIT
![image](https://github.com/transcom/mymove/assets/136814362/c1905e26-a5c3-4227-b133-309e40a39c25)
